src/Presenters/Content/WikiContentProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Presenters\Content;
```

Comment:
I'm fine with this being merged as-is btw. We can move this stuff around further afterwards.